salt/modules/win_file.py

```diff
@@ -42,6 +42,9 @@
 # pylint: enable=W0611
 
 # Import salt libs
+=======
```

Comment:
This line must be removed before we can get this in. This is likely letf-over from the merge-conflict.